specs/kyber/tests/interop_with_libcrux.rs

```diff
@@ -0,0 +1,93 @@
+//! Test spec - code interop
+
+use hacspec_kyber::{KYBER768_KEY_GENERATION_SEED_SIZE, KYBER768_SHARED_SECRET_SIZE, KYBER768_CIPHERTEXT_SIZE};
+use rand::{rngs::OsRng, RngCore};
+use libcrux::kem::KyberCiphertext;
+
+#[test]
+fn same_inputs_result_in_same_output() {
+    let mut keygen_seed = [0u8; KYBER768_KEY_GENERATION_SEED_SIZE];
+    OsRng.fill_bytes(&mut keygen_seed);
+
+    let libcrux_key_pair_result = libcrux::kem::kyber768_generate_keypair_derand(keygen_seed);
+    let spec_key_pair_result = hacspec_kyber::generate_keypair(keygen_seed);
+
+    if libcrux_key_pair_result.is_err() {
+        spec_key_pair_result.expect_err("If rejection sampling failed in libcrux, it should fail in the spec as well.");
+        return
```

Comment:
Let's not return here, but try again. It's very unlikely that this happens. But we don't want this test to always get out here.